assets/css/appV2/table.scss

```diff
@@ -12,6 +12,24 @@ th:last-child{
   border-radius: 0 10px 10px 0;
 }
 
+.personal-data-card::before {
+  content: '';
+  position: absolute;
+  left: 0;
+  top: 0;
+  height: 100%;
+  width: 5px;
+  border-radius: 0 0 0 30px;
```

Comment:
Maybe i'm wrong, but does it improve left bar render ?  `border-radius: 30px 0 0 30px;`

Reply:
I made it like that because I followed the canva https://www.canva.com/design/DAGDaOMZSew/bl7k15_4ANmyPUUGLoDk5g/edit

Reply:
Oh my bad, i love radius on borders